specification/databricks/resource-manager/Microsoft.Databricks/stable/2024-05-01/accessconnector.json

```diff
@@ -0,0 +1,438 @@
+            "schema": {
+              "$ref": "#/definitions/AccessConnectorListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Databricks/accessConnectors": {
+      "get": {
+        "tags": [
+          "AccessConnector"
+        ],
+        "operationId": "AccessConnectors_ListBySubscription",
+        "description": "Gets all the azure databricks accessConnectors within a subscription.",
+        "x-ms-examples": {
+          "Lists all the azure databricks accessConnectors within a subscription.": {
+            "$ref": "./examples/AccessConnectorsListBySubscriptionId.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns an array of azure databricks accessConnectors.",
+            "schema": {
+              "$ref": "#/definitions/AccessConnectorListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AccessConnector": {
+      "type": "object",
+      "properties": {
+        "identity": {
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentity"
+        },
+        "systemData": {
+          "description": "The system metadata relating to this resource",
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData"
+        },
+        "properties": {
+          "description": "Azure Databricks accessConnector properties",
+          "$ref": "#/definitions/AccessConnectorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../preview/2022-04-01-preview/databricks.json#/definitions/TrackedResource"
+        }
+      ],
+      "description": "Information about azure databricks accessConnector."
+    },
+    "AccessConnectorProperties": {
+      "type": "object",
+      "properties": {
+        "provisioningState": {
+          "description": "Provisioning status of the accessConnector.",
```

Comment:
>accessConnector [](http://example.com/codeflow?start=53&length=15)

nit for your docs -- try to use plain English style; as in "the access connector"

Reply:
@curiekim0 can you update the description as per @TimLovellSmith's suggestion.

Reply:
Updated all instances of "accessConnector" to "Access Connector"